pa11ycrawler/settings.py

```diff
@@ -101,5 +19,15 @@
 # See http://scrapy.readthedocs.org/en/latest/topics/item-pipeline.html
 ITEM_PIPELINES = {
     'pa11ycrawler.pipelines.DuplicatesPipeline': 200,
+    'pa11ycrawler.pipelines.DropDRFPipeline': 250,
     'pa11ycrawler.pipelines.Pa11yPipeline': 300,
 }
+
+# Other items you are likely to want to override ---------------
+CONCURRENT_REQUESTS = 16
```

Comment:
Ok great. I wonder if we could do some optimizations here, then, so we can shorten it to the point where it can go into PR builds. What do you think? (Obviously separate from this PR, but...)